### ISSUE TITLE
support for "static" record marshalling when not using the BeanWriter.write

### ISSUE THREAD

**Issue by GoogleCodeExporter** [unlockha/beanio#79]
```
This is basically the same capability request as Issue 36 (Add support for 
"static" record marshalling).  However, the solution to Issue 36 won't work for 
me because I can't use BeanWriter.write.  

Stated more generally, I need to be able to marshal a "static" record not bound 
to a bean object in order to write out the header row in a file.

I need "Marshaller.marshal("header", null)" rather than 
"BeanWriter.write("header", null)".


```

Original issue reported on code.google.com by `the.stri...@gmail.com` on 5 Jun 2013 at 8:43

**Comment by GoogleCodeExporter** [comment]
```
That makes sense, I'll look into it for 2.1.0.M2.

Thanks,
Kevin
```

Original comment by `kevin.s...@gmail.com` on 6 Jun 2013 at 2:08
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Fixed for 2.1.0.M2.  Snapshot JAR attached.
```

Original comment by `kevin.s...@gmail.com` on 14 Jul 2013 at 5:41
- Added labels: **Milestone-2.1.0.M2**

Attachments:
- [beanio-2.1.0.SNAPSHOT.jar](https://storage.googleapis.com/google-code-attachments/beanio/issue-79/comment-2/beanio-2.1.0.SNAPSHOT.jar)

**Comment by GoogleCodeExporter** [comment]
Original comment by `kevin.s...@gmail.com` on 16 Nov 2013 at 5:29
- Changed state: **Fixed**
